SECURITY.md

```diff
@@ -0,0 +1,122 @@
+
+* **The GitHub security advisory.** These advisories will let us collaborate in private, and they are appropriate in cases where disclosing the vulnerability prior to remediating it could put our users or developers at risk.
+* **(Optionally) An additional GitHub issue and pull request.** This will immediately disclose the vulnerability, and we will take this approach when immediate disclosure poses little risk to our users and developers. For example, when the vulnerability is already publicly known. Unlike security advisories, CI checks can run on these PRs.
+
+Note that we should create a GitHub security advisory even if we choose to collaborate on a fix using an open issue or PR because we’ll want to publish the advisory when disclosing the vulnerability.
+
+### Request a CVE
+
+A CVE should be requested for all security vulnerabilities. Since we create a GitHub Security Advisory, we can [get a CVE from GitHub](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#cve-identification-numbers). As a backup, we can [get a CVE from MITRE](https://cveform.mitre.org/).
+
+### Develop a Patch
+
+Regardless of which approach we take, the VRT will prioritize developing a patch to fix the vulnerability. If the vulnerability is easily exploitable, remediation should take priority over all other work. Remediation should be completed within 7 days of the report being classified as a high-severity vulnerability.
+
+## Disclose
+
+We generally follow a 90-day disclosure timeframe, meaning that we ask that reporters give us 90 days to fix vulnerabilities before publicly disclosing them. 90 days should be viewed as an upper bound, and we should aim to remediate vulnerabilities as quickly as possible. In all cases, the disclosure timeline should be explicitly coordinated with the reporter, and we should arrange to publicly disclose the vulnerability simultaneously with the reporter. Our disclosure will include credit for the reporter if they so wish.
+
+In rare cases, it may be appropriate to delay public disclosure even after the patch has been published and deployed. However, since our source code is public, we should assume that attackers will quickly reverse-engineer the vulnerability from our patch and so err on the side of disclosing early.
+
+Our public disclosure should take the form of a published GitHub Security Advisory. Here is a template:
+
+```
+# Security advisory for $CVEID
+
+## Summary
+
+<!--
+A brief (as short as possible, about a paragraph) summary of the vulnerability using technical details. The goal of this is to allow the reader to do a quick assessment of what the bug is about.
+-->
+
+* **CVE**: $CVE-NUMBER
+* **Affected versions**: $VERSIONS
+* **Reporter**: $NAME $AFFILIATION
+
+## Severity
+<!--
+[Low, Medium, HIGH, **CRITICAL**] - Accompany your assessment with a motivation, and even a good attack scenario to explain the risk associated with the vulnerability.
```

Comment:
How does one fill this out? It doesn't match our internal tiers for severity, so it's not exactly clear to me how this should map over.

Reply:
These are determined based on [CVSS](https://www.first.org/cvss/user-guide). I'll update the comment to explain